gem2rpm/foreman_plugin.spec.erb

```diff
@@ -0,0 +1,184 @@
+# FIXME:
+#   1. Change scl_prefix to _ruby or _ror for Requires/BuildRequires that are
+#      in Ruby or Rails software collections (not Foreman's)
+#   2. Edit foreman requirement(s) and specify minimum version
+#   3. Delete these lines
+#
+# Generated from <%= package.spec.file_name %> by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name <%= spec.name %>
+%global plugin_name <%= spec.name.sub(/\Aforeman_/, '') %>
+<%
+has_assets = spec.files.any? { |f| f.start_with?('app/assets/') }
+has_apidoc = spec.files.any? { |f| f =~ %r{\Aapp/controllers/?.*/api/} }
+has_db_migrations = spec.files.any? { |f| f.start_with?('db/migrate/') }
+has_db_seeds = spec.files.any? { |f| f.start_with?('db/seeds.d/') }
+
+precompile_flags = ''
+precompile_flags << ' -a' if has_apidoc
+precompile_flags << ' -s' if has_assets
+
+config.rules[:doc] << /\/?CHANGES/
+config.rules[:ignore] << /.tx(\/.*)?/
+-%>
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: <%= spec.version %>
+Release: 1%{?dist}
+Summary: <%= spec.summary.gsub(/\.$/, "") %>
+Group: Applications/Systems
+License: <%= spec.licenses.join(" and ") %>
+<% if spec.homepage -%>
+URL: <%= spec.homepage %>
+<% end -%>
+Source0: <%= download_path.sub('https:', 'http:') %>%{gem_name}-%{version}.gem
+Requires: foreman >= FIXME
+Requires: %{?scl_prefix_ruby}ruby(release)
+<% for req in spec.required_ruby_version -%>
+Requires: %{?scl_prefix_ruby}ruby <%= req %>
+<% end -%>
+<% for req in spec.required_rubygems_version -%>
+Requires: %{?scl_prefix_ruby}ruby(rubygems) <%= req %>
+<% end -%>
+<% for d in spec.runtime_dependencies -%>
+<% for req in d.requirement -%>
```

Comment:
Just a nitpick but maybe you can indent the ruby code, and leave the actual spec code unindented? It gets a bit difficult to read IMO